### PR TITLE
ts/bb-indicator

### PR DESCRIPTION
**Pull request by karolkolodziej**

### CODE REVIEW

File: ts/Stock/Indicators/BB/BBIndicator.ts [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+    *
+    * */
+
+    public data: Array<BBPoint> = void 0 as any;
+
+    public options: BBOptions = void 0 as any;
+
+    public points: Array<BBPoint> = void 0 as any;
+
+    public init(this: BBIndicator): void {
```
Comment (bre1470) [inline ts/Stock/Indicators/BB/BBIndicator.ts:153]: The generic comment is incorrect. You have two options to solve this. Remove this generic comment, or replace it with "Properties" and add "Functions". In a class pattern property assignments go on the instance and only functions go on the prototype.

**Review by bre1470 — CHANGES_REQUESTED** [review]
Thank you! Please note the review comment.

**Review by bre1470 — APPROVED** [review]
Looks good. 👍

**Comment by TorsteinHonsi** [comment]
Pending tests after conflict resolving, otherwise ready to merge.

**Pull request merged**